packages/core/misc-widgets/src/components/github-star/README.md

```diff
@@ -0,0 +1,24 @@
+# Github Star
+
+A Github Star component for displaying number of stars received by the repo on Github
+
+- [Features](#features)
+- [Usage](#usage)
+  - [Props](#props)
+
+## Features
+
+- Render number of stars received by the repo on Github
+- Customize the repo url that opens in a new tab upon clicking this component
+
+## Usage
+
+### Props
+
+#### `url`
+
+- type: `String`
+- required: `false`
```

Comment:
Sorry, I just meant updating this docs item
```suggestion
- required: `true`
```

Reply:
Whoops, updated 👍